### DISCUSSION TITLE
Remove prepare script?

### DISCUSSION THREAD

**Discussion by gtb104** [typicode/husky#1199] [Q&A] [answered]
Once you've run `npm run prepare`, do we still need to keep that script around? It seems like a single-run kind of script.

**Comment by TheKnarf** [comment]
It is as far as I know.

Personally I added the following to my `package.json`'s script section:

```json
 "scripts": {
    "husky:enable": "husky",
    "husky:disable": "git config --unset core.hooksPath",
 }
```

And I feel like that should be the default, that way I can enable and disable husky at will.

**Comment by typicode** [comment, marked as answer]
Both options are valid. `prepare` for automatic install or custom scripts for opt-in.
